### DISCUSSION TITLE
Messaging Android App coding

### DISCUSSION THREAD

**Discussion by Dajlistic** [community/community#21373] [Other Feature Feedback, Questions, & Ideas]
<p>Hello guys<br>
Am new here and not verse with Java. Please can someone help me on how to start?</p>

**Comment by AndreaGriffiths11** [comment]
<p>We are happy you are here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>If you have no knowledge of Java, then you can consider an alternative: Kotlin. Google is encouraging people to use Kotlin as a viable alternative to Java for Android.</p>
<p>Either Java or Kotlin: get a copy of the IntelliJ IDE (free version). Read the docs and set up a basic project, get it running in an emulation environment. Since you’re talking about messaging, you probably need a server, database, and API to talk to it.  So you need to figure that out as well.</p>

**Comment by aprildawson** [comment]
<p>Live button on facebook</p>
